server/pom.xml

```diff
@@ -183,7 +183,7 @@
                             <jvmFlag>-server</jvmFlag>
                             <jvmFlag>-Xss512k</jvmFlag>
                             <jvmFlag>-XX:+UnlockExperimentalVMOptions</jvmFlag>
-                            <jvmFlag>-XX:+UseCGroupMemoryLimitForHeap</jvmFlag>
+                            <jvmFlag>-XX:+UseContainerSupport</jvmFlag>
```

Comment:
since  jdk11+

Reply:
Actually, the arg is added in Java 8u191 also, you can reference [
JDK 8u191 Update Release Notes](https://www.oracle.com/technetwork/java/javase/8u191-relnotes-5032181.html), our image is based on 8u212, so it can execute correctly.  And I had a try of image, it works well.

![image](https://user-images.githubusercontent.com/13172836/70390042-20c10980-1a02-11ea-9c95-a5be82b10c10.png)
![image](https://user-images.githubusercontent.com/13172836/70390047-2585bd80-1a02-11ea-90ca-f4335126bb6f.png)